lib/sidekiq.rb

```diff
@@ -106,19 +106,17 @@ def self.redis
 
   def self.redis_info
     redis do |conn|
-      begin
-        # admin commands can't go through redis-namespace starting
-        # in redis-namespace 2.0
-        if conn.respond_to?(:namespace)
-          conn.redis.info
-        else
-          conn.info
-        end
-      rescue Redis::CommandError => ex
-        #2850 return fake version when INFO command has (probably) been renamed
-        raise unless ex.message =~ /unknown command/
-        FAKE_INFO
+      # admin commands can't go through redis-namespace starting
+      # in redis-namespace 2.0
+      if conn.respond_to?(:namespace)
+        conn.redis.info
+      else
+        conn.info
       end
+    rescue Redis::CommandError => ex
```

Comment:
I am not sure this syntax was supported by all Ruby versions supported by Sidekiq.

Reply:
It's this feature was merge in the 2.5 Ruby version. And Sidekiq 6.0 support only Ruby >= 2.5

Ref: https://bugs.ruby-lang.org/issues/7882